gdx/src/com/badlogic/gdx/scenes/scene2d/Stage.java

```diff
@@ -110,8 +110,7 @@ public Stage (Viewport viewport, Batch batch) {
 		this.viewport = viewport;
 		this.batch = batch;
 
-		root = new Group();
```

Comment:
We have different opinions on that. If this is overriden in a subclass, then the subclass is intended to do have a different behaviour. 
Invoking overridable methods is done everywhere in scene2d actors.

Reply:
On one hand, I agree that it can cause bugs.
But on the other hand, not calling it would force anything overriding the function to call it explicitly.